code/styles.css

```diff
@@ -0,0 +1,80 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+.top {
+  padding: 20px;
+  background-image: url(images/algues.jpeg);
+  /*background-color: rgb(144, 232, 144);*/
+  font-style: white;
+  /*font-display: WHITE;*/
+  /*color: rgb(241, 245, 249);*/
+}
+
+.middle {
+  background-color: white;
+}
+
+.down {
+  background-color: #90e890;
+}
+
+img {
+  border-radius: 50%;
+}
```

Comment:
If you want an image to be truly round, you need to first make it square so that it has the same height and width. In case of distortion, you can always add object-fit: cover; to solve that.